src/utils/animations.ts

```diff
@@ -1,7 +1,8 @@
 import { RefObject } from 'react'
 
 export function isAnimating(node: Animatable | Document) {
-  return (node.getAnimations().length ?? 0) > 0
+  // ?. used here to support tests that call this function
+  return (node?.getAnimations?.().length ?? 0) > 0
```

Comment:
see comment

Reply:
Which tests pass undefined, here?
This should be codified in the parameter, with the comment there:

    // Some tests will pass undefined for the Document.
    export function isAnimating(node?: Animatable | Document) {